__mocks__/patient-registration.mock.ts

```diff
@@ -1,4 +1,47 @@
-import { AddressTemplate } from '../../../patient-registration.types';
+type AddressProperties =
```

Comment:
It's fine, but it would probably be better if we didn't have to duplicate the types here? I'm guessing we can't just import them in `__mocks__`